src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/OutputRow.tsx

```diff
@@ -90,6 +90,9 @@ export const OutputRow = ({
       removeItemDescription = t('registration.resource_type.artistic.remove_announcement_description', { name: title });
       break;
     case 'Venue':
```

Comment:
Hvis vi skal vise samme her som for alle andre typene over så er det bedre å flytte `case 'Venu'` opp dit, og gjenbruke det som allerede ligger der.
Antar vi ønsker det samme for `PerformingArtsVenue` også, eller? I så fall trenger vi vel ikke lenger noen `switch` heller, da alle typene skal vise det samme anyways